### ISSUE TITLE
[Bug] "Select all" disabled

### ISSUE THREAD

**Issue by KaKi87** [florisboard/florisboard#2176]
#### Short description
Sometimes when clicking an input, the *select all* action is disabled while partial selection (*backspace* key swipe) still works, which usage enables the former.

The bug doesn't always occur and I can't tell why.

#### Steps to reproduce
1. Click an input
2. Notice *select all* disabled
3. Perform partial selection
4. Notice *select all* enabled

#### Environment information
- FlorisBoard Version: 0.4.0-alpha.4+90
- Install Source: F-Droid
- Device: Xiaomi Mi 8
- Android: 13, LineageOS

---
Thanks

**Comment by KLaurence** [comment]
Might be related with #2024 and #2097.

**Comment by KaKi87** [comment]
Hello,
Any news on this ?
Thanks

**Comment by firefoxlover** [comment]
Are you referring to "the select all button does not work"?

I dont think it is disabled it just doesnt work in certain apps.

- Obtainium input fields
- Vanadium, Mull input in some websites

**Comment by KaKi87** [comment]
> it just doesnt work in certain apps

That's not my case.

See step 4 : the button ends up working.

**Comment by maivohun** [comment]
Also happens when using Firefox (beta, nightly), Cromite, Element (X) and SchildiChat (Next). I am able to reproduce this easily when using Firefox or Cromite, and GitHub:

https://github.com/user-attachments/assets/36ddf428-16c2-438e-b674-141a57aa4b5f

Original recorded video: https://files.catbox.moe/g4x0uj.mp4

Only way that I have found for it to make work without switching fields or refocusing, is to press "show/hide additional actions" key.

**Comment by rancidfrog** [comment]
Latest stable releases causes select all button to not work more often.
Before, there were less cases where it did not work, now it is annoying

**Comment by maivohun** [comment]
> Before, there were less cases where it did not work, now it is annoying

@rancidfrog Agreed! That's why I mentioned the workaround that has worked for me all the time (so far). Just tap the "show/hide additional actions" key:

![ResizedImage_2024-09-28_13-08-05_1857](https://github.com/user-attachments/assets/51b1cd1d-7341-4298-a7e6-be775f0ba656)

**Comment by slimfish08** [comment]
> > Before, there were less cases where it did not work, now it is annoying
> 
> @rancidfrog Agreed! That's why I mentioned the workaround that has worked for me all the time (so far). Just tap the "show/hide additional actions" key:
> 
> ![ResizedImage_2024-09-28_13-08-05_1857](https://private-user-images.githubusercontent.com/182167554/371740083-51b1cd1d-7341-4298-a7e6-be775f0ba656.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA0NzExMDYsIm5iZiI6MTczMDQ3MDgwNiwicGF0aCI6Ii8xODIxNjc1NTQvMzcxNzQwMDgzLTUxYjFjZDFkLTczNDEtNDI5OC1hN2U2LWJlNzc1ZjBiYTY1Ni5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTAxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEwMVQxNDIwMDZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jNzNkMzMzYjI3NzEzNzE5NjNhMGYzYjcyOGY0ZjAyOTgzYzY3Y2Q3ZmE3MjZjMWI4OTdkN2E0MzdmZjEwODFjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.7kLTXOXi10spb7rTpqQAKRv_Ehu56yqa69IzzbqbQIU)


https://github.com/user-attachments/assets/51cd38c0-2082-4adf-b53f-15101b8d18a6

The workaround works for me as well!

**Comment by int0x25** [comment]
Opening the clipboard history also reactivates the button. On Cromite, I'm able to consistently trigger the gray-out by visiting a page from the URL bar.

Now that I think about it, I don't really know what this button's intended behavior is. When it's enabled, it's enabled even when the text field is empty, and when it's disabled, it's disabled even when there's text.

**Comment by Zokii0** [comment]
Is there any update on this? I'm also facing this issue for longer time (year or so), and seems like swipe to delete is also affected at same time. Why in FlorisBoard select all never worked properly, while in HeliBoard for example always worked?

**Comment by mooreye** [comment]
For me, if select all button is in the bar, its greyed out until I tap the 3 dots. If the button is in the 3 dots section, it seems to always be available
